hack/build/repos.bzl

```diff
@@ -2930,3 +2930,27 @@ def go_repositories():
         sum = "h1:YXTMot5Qz/X1iBRJhAt+vI+HVttY0WkSqqhKxQ0xVbA=",
         version = "v4.0.1",
     )
+    go_repository(
+        name = "com_github_openshift_api",
+        build_file_generation = "on",
+        build_file_proto_mode = "disable",
+        importpath = "github.com/openshift/api",
+        sum = "h1:PbIRNLKSDnc+P0XT67emLMAItK41ntNYs5K8OUtJQ6U=",
+        version = "v0.0.0-20200330134433-8e259f67fc55",
```

Comment:
Does this repo have release tags? Would make the maintaining of the deps more easy for us

Reply:
It does not https://github.com/openshift/api/tags